nilearn/input_data/fmriprep_confounds.py

```diff
@@ -98,19 +103,39 @@ def fmriprep_confounds(img_files,
         - `func.gii`: list of a pair of paths to files, optionally as a list
           of lists.
 
-    strategy : list of strings, default ["motion", "high_pass", "wm_csf"]
+    strategy : tuple or list of strings.
+        Default ("motion", "high_pass", "wm_csf")
         The type of noise components to include.
 
-        - "motion":  head motion estimates.
-        - "high_pass" discrete cosines covering low frequencies.
+        - "motion":  head motion estimates. Associated parameter: `motion`
         - "wm_csf" confounds derived from white matter and cerebrospinal fluid.
-        - "global" confounds derived from the global signal.
+          Associated parameter: `wm_csf`
+        - "global_signal" confounds derived from the global signal.
+          Associated parameter: `global_signal`
         - "compcor" confounds derived from CompCor :footcite:`BEHZADI200790`.
+          When using this noise component, "high_pass" must also be applied.
+          Associated parameter: `compcor`, `n_compcor`
         - "ica_aroma" confounds derived from ICA-AROMA :footcite:`Pruim2015`.
+          Associated parameter: `ica_aroma`
         - "scrub" regressors for :footcite:`Power2014` scrubbing approach.
+          Associated parameter: `scrub`, `fd_threshold`, `std_dvars_threshold`
+
+        For each component above, associated parameters will be applied if
+        present. Otherwise, any values supplied to the parameters are ignored.
```

Comment:
Unlcearl what "otherwise" referes to here. Could be "if associated parameters are not present" or could be that the components is not specified...